packages/dashboard/src/CdcDashboardComponent.tsx

```diff
@@ -765,7 +776,7 @@ export default function CdcDashboard({ initialState, isEditor = false, isDebug =
 
   return (
     <GlobalContextProvider>
-      <DashboardContext.Provider value={{ ...state, setParentConfig: editorContext.setTempConfig, outerContainerRef, isDebug }}>
+      <DashboardContext.Provider value={{ ...state, setParentConfig: editorContext.setTempConfig, outerContainerRef, isDebug, isPreview }}>
```

Comment:
if you put isPreview here you don't need to also pass it as a prop on 691. VisualizationRow already is hooked into the DashboardContext. so pull it from there.